src/status_im2/subs/contact.cljs

```diff
@@ -251,4 +251,32 @@
                (assoc acc address contact)
                acc))
            {}
-           contacts)))
\ No newline at end of file
+           contacts)))
+
+(re-frame/reg-sub
+ :contacts/filtered-active-sections
+ :<- [:contacts/active]
+ :<- [:contacts/search-query]
+ (fn [[contacts query]]
+   (let [data (atom {})]
```

Comment:
@OmarBasem, I don't know if it helps in the midst of so many comments, but this is the only outstanding thread left for me to approve your PR.

I've noticed that lots of layer-3 subscriptions don't have tests yet, and that it might not be obvious how to test them given they're registered as anonymous functions in an internal state managed by re-frame. Please, ignore my comment about testing them. I think this kind of test requires a separate PR example in order to raise awareness and share knowledge, so it would be too much effort for this PR, which is already quite busy ;)

Reply:
You are right @ilmotta. 
This PR has gotten 80+ reviews/comments :'(